test/e2e/testdata/aemm-config-integ.json

```diff
@@ -21,7 +21,7 @@
         "ramdiskId": null,
         "kernelId": null,
         "devpayProductCodes": null,
-        "marketplaceProductCodes": null,
+        "marketplaceProductCodes": ["4i20ezfza3p7xx2kt2g8weu2u"],
```

Comment:
Although a list, I avoided this case because I don't think there could ever be more than 1 item in reality. From the [docs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-identity-documents.html):

![image](https://user-images.githubusercontent.com/61433408/175988677-b10cabd2-c3e0-46af-a37a-653266248665.png)


If you know of any use cases with multiple codes and/or if it is possible to have such, then could you share the details? I'm reading into [product codes](https://docs.aws.amazon.com/marketplace/latest/userguide/ami-getting-started.html) now in case I misinterpreted the IMDS description.

Reply:
I am not aware of any cases with multiple codes (due to lack of experience in the area). However, reading this example in the link you shared makes me think that the new AMI could end up with two codes (just a guess) - `Product codes are propagated automatically as buyers work with the software. For example, a customer subscribes and launches an AMI, configures it, and produces a new AMI. The new AMI still contains the original product code, so correct usage tracking and permissions remains in place.`

Irrespective, since we are supporting a list of strings along with some formatting logic in a mock API, it makes sense to test the scenario, IMO.

Reply:
That entry just says an updated AMI will inherit a product code not that another one is possible. Sure, are okay with a unit test?